tuichain/api/views/loans.py

```diff
@@ -320,18 +321,28 @@ def get_loan_request(request, id):
             status=HTTP_404_NOT_FOUND,
         )
 
+    loan_dict = loan.to_dict()
+
+    if loan.state == 2:
```

Comment:
Would be preferable to use something like `LoanState(loan.state) == LoanState.APPROVED`. Ditto for other occurrences.